portality/view/status.py

```diff
@@ -1,10 +1,14 @@
+import json
+import math
+import os
+import time
+
+import requests
 from flask import Blueprint, make_response, url_for
+
 from portality import util
+from portality.bll import background_task_status
```

Comment:
We have a pattern for accessing the BLL, we should access it like

```
from portality.bll import DOAJ
background_service = DOAJ.backgroundService()
```

Reply:
for more information, 
* any other benefit to create services factory, except avoid Circular import problem? 
* is it because `pre-loading` service is  too slow, if yes, how slow could it be?
* if I should add `background_task_status` to bll.doaj.py, could I avoid to create a class as below ?
```
@classmethod
def backgroundTaskStatus(cls):
    from portality.bll.services import background_task_status
    return background_task_status
```

Reply:
Avoiding circular imports and convenience for mocking during testing are the primary motivations.  It's not the only pattern we could use for that, but for now it's the one that we landed on.  I'm not averse to doing it differently, but if we change it I want it changed for everything so that we have a consistent pattern.